libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/SequenceContext.cs

```diff
@@ -6,12 +6,15 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Bot.Builder.Dialogs.Adaptive.Input;
 
 namespace Microsoft.Bot.Builder.Dialogs.Adaptive
 {
     public class SequenceContext : DialogContext
     {
+#pragma warning disable SA1310 // Field should not contain underscore.
+        private const string WAIT_FOR_INPUT_PATH = "turn.waitForInput";
```

Comment:
should this be on TurnPath?

Reply:
Great discussion for master--I didn't define this, just the pragmas.

---
In reply to: [343916844](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r343916844) [](ancestors = 343916844)

Reply:
probably Pascal case this.

---
In reply to: [343938776](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r343938776) [](ancestors = 343938776,343916844)